<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="7" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Ant suite" duration-ms="0" started-at="2021-03-17T10:52:13 MSK" finished-at="2021-03-17T10:52:13 MSK">
    <groups>
    </groups>
    <test name="Ant test" duration-ms="0" started-at="2021-03-17T10:52:13 MSK" finished-at="2021-03-17T10:52:13 MSK">
    </test> <!-- Ant test -->
  </suite> <!-- Ant suite -->
  <suite name="TestNG Tests" duration-ms="3200" started-at="2021-03-17T10:52:10 MSK" finished-at="2021-03-17T10:52:13 MSK">
    <groups>
    </groups>
    <test name="Department tests" duration-ms="3200" started-at="2021-03-17T10:52:10 MSK" finished-at="2021-03-17T10:52:13 MSK">
      <class name="test.PositionTest">
        <test-method status="PASS" signature="loadPosition()[pri:0, instance:test.PositionTest@59fa1d9b]" name="loadPosition" duration-ms="15" started-at="2021-03-17T10:52:13 MSK" finished-at="2021-03-17T10:52:13 MSK">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadPosition -->
      </class> <!-- test.PositionTest -->
      <class name="test.StaffMemberTest">
        <test-method status="PASS" signature="addStaffMember()[pri:0, instance:test.StaffMemberTest@d8355a8]" name="addStaffMember" duration-ms="16" started-at="2021-03-17T10:52:12 MSK" finished-at="2021-03-17T10:52:12 MSK">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addStaffMember -->
        <test-method status="PASS" signature="deleteStaffMember()[pri:0, instance:test.StaffMemberTest@d8355a8]" name="deleteStaffMember" duration-ms="293" started-at="2021-03-17T10:52:12 MSK" finished-at="2021-03-17T10:52:13 MSK">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteStaffMember -->
        <test-method status="FAIL" signature="getStaffMembersByDepartment()[pri:0, instance:test.StaffMemberTest@d8355a8]" name="getStaffMembersByDepartment" duration-ms="116" started-at="2021-03-17T10:52:13 MSK" finished-at="2021-03-17T10:52:13 MSK">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at test.StaffMemberTest.getStaffMembersByDepartment(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.TestNG.privateMain(TestNG.java:1365)
at org.testng.TestNG.main(TestNG.java:1329)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStaffMembersByDepartment -->
        <test-method status="PASS" signature="updateStaffMember()[pri:0, instance:test.StaffMemberTest@d8355a8]" name="updateStaffMember" duration-ms="18" started-at="2021-03-17T10:52:13 MSK" finished-at="2021-03-17T10:52:13 MSK">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateStaffMember -->
      </class> <!-- test.StaffMemberTest -->
      <class name="test.EmployeeTest">
        <test-method status="PASS" signature="loadEmployee()[pri:0, instance:test.EmployeeTest@28d25987]" name="loadEmployee" duration-ms="17" started-at="2021-03-17T10:52:13 MSK" finished-at="2021-03-17T10:52:13 MSK">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadEmployee -->
      </class> <!-- test.EmployeeTest -->
      <class name="test.DepartmentTest">
        <test-method status="PASS" signature="loadDepartmentTest()[pri:0, instance:test.DepartmentTest@8bd1b6a]" name="loadDepartmentTest" duration-ms="2667" started-at="2021-03-17T10:52:10 MSK" finished-at="2021-03-17T10:52:12 MSK">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadDepartmentTest -->
      </class> <!-- test.DepartmentTest -->
    </test> <!-- Department tests -->
  </suite> <!-- TestNG Tests -->
</testng-results>
